dev/builder/build.sh

```diff
@@ -57,7 +57,7 @@ echo "Starting CKBuilder..."
 
 JAVA_ARGS=${ARGS// -t / } # Remove -t from args.
 
-VERSION=`node -pe "require('./../../package.json').version"`
+VERSION=$(cat ./../../package.json | grep \"version\"\: | awk -F: '{ print $2}' | sed 's/[\",]//g' | tr -d '[[:space:]]')
```

Comment:
This works fine, but maybe it will be simpler and more readable to use something like:

`grep '"version":' package.json | sed $'s/[\t\",: ]//g; s/version//g'`

it greps the version line and then replaces/removes all unnecessary characters and `version` string which leaves only a version number.